src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -319,7 +319,7 @@
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
     <Field Name="Operand" Type="BoundExpression"/>
     <!-- Well-known member populated during lowering -->
-    <Field Name="GetTypeFromHandle" Type="MethodSymbol" Null="allow"/>
+    <Field Name="GetTypeFromHandle" Type="MethodSymbol?"/>
```

Comment:
>GetTypeFromHandle [](start = 17, length = 17)

not blocking this PR: should this be `Opt`?

Reply:
I don't think so. I previously used `Opt` as a suffix to indicate nullability because we didn't have nullability. Now that we do, I think we should phase out use of it in our names.